test/end-to-end/test_studio_when_ssl_cert_file_isnt_cached.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# Test that when SSL_CERT_FILE isn't cached, the variable isn't 
+# set on the inside of the studio.  
+# https://github.com/habitat-sh/habitat/issues/7219
+
+set -euo pipefail
+
+source .expeditor/scripts/end_to_end/shared_end_to_end.sh
+ 
+echo "--- Generating a signing key"
+hab origin key generate "$HAB_ORIGIN"
+
+echo "--- Removing any existing cached certificates"
+rm -f /hab/cache/ssl/*
+rm -f ~/.hab/cache/ssl/*
+
+# This must be run immediatly before `studio run`, as 
+# the auto-installation of the studio on first-run triggers
+# the behavior we're attempting to test
+echo "--- Uninstalling any existing studio packages"
+hab pkg uninstall core/hab-studio || true
```

Comment:
Is there any danger of there being multiple `core/hab-studio` packages on disk? `hab pkg uninstall` only removes the most recent one, if it's there.

(:thinking: I wonder if a new `hab pkg uninstall core/hab-studio --all` flag is warranted? :thinking:)

Reply:
I thought about this and convinced myself that, in CI at least, it should be ok. Worst case remaining studios would be from an older version which _shouldn't_ be used.    

If we're feeling extra paranoid, probably a good thing when testing the studio, I can wrap this in a while loop to remove all studios, or just `rm -rf /hab/pkgs/core/hab-studio` and call it a day.

Reply:
Wrapping it in a while loop seems like a good way forward, as that stays true to the "don't rely on internal implementation details" spirit behind `hab pkg uninstall`. It also makes it very easy to modify in the future if / when we ever do get an `--all` option.

Reply:
Resolved in f1788cdb27b832a29929efb07192f36a6350ce9e